web/i18n/language.ts

```diff
@@ -26,96 +26,119 @@ export const languages = [
   {
     value: 'en-US',
     name: 'English (United States)',
+    prompt_name: 'English',
     example: 'Hello, Dify!',
     supported: true,
   },
   {
     value: 'zh-Hans',
     name: '简体中文',
+    prompt_name: 'Chinese',
     example: '你好，Dify！',
     supported: true,
   },
   {
     value: 'zh-Hant',
     name: '繁體中文',
+    prompt_name: 'Chinese',
     example: '你好，Dify！',
     supported: true,
   },
   {
     value: 'pt-BR',
     name: 'Português (Brasil)',
+    prompt_name: 'Portuguese',
     example: 'Olá, Dify!',
     supported: true,
   },
   {
     value: 'es-ES',
     name: 'Español (España)',
+    prompt_name: 'Spanish',
     example: 'Saluton, Dify!',
     supported: false,
   },
   {
     value: 'fr-FR',
     name: 'Français (France)',
+    prompt_name: 'French',
     example: 'Bonjour, Dify!',
     supported: true,
   },
   {
     value: 'de-DE',
     name: 'Deutsch (Deutschland)',
+    prompt_name: 'German',
     example: 'Hallo, Dify!',
     supported: true,
   },
   {
     value: 'ja-JP',
     name: '日本語 (日本)',
+    prompt_name: 'Japanese',
     example: 'こんにちは、Dify!',
     supported: true,
   },
   {
     value: 'ko-KR',
     name: '한국어 (대한민국)',
+    prompt_name: 'Korean',
     example: '안녕, Dify!',
     supported: false,
   },
   {
     value: 'ru-RU',
     name: 'Русский (Россия)',
+    prompt_name: 'Russian',
     example: ' Привет, Dify!',
     supported: false,
   },
   {
     value: 'it-IT',
     name: 'Italiano (Italia)',
+    prompt_name: 'Italian',
     example: 'Ciao, Dify!',
     supported: false,
   },
   {
     value: 'th-TH',
     name: 'ไทย (ประเทศไทย)',
+    prompt_name: 'Thai',
     example: 'สวัสดี Dify!',
     supported: false,
   },
   {
     value: 'id-ID',
     name: 'Bahasa Indonesia',
+    prompt_name: 'Indonesian',
     example: 'Saluto, Dify!',
     supported: false,
   },
   {
     value: 'uk-UA',
     name: 'Українська (Україна)',
+    prompt_name: 'Ukrainian',
     example: 'Привет, Dify!',
     supported: true,
   },
   {
     value: 'vi-VN',
     name: 'Tiếng Việt (Việt Nam)',
+    prompt_name: 'Vietnamese',
     example: 'Xin chào, Dify!',
     supported: true,
   },
+  {
```

Comment:
If you add a new language here, you will need to add the whole language folder and missing languages in the folder, because it will take it account into the initialization steps.

https://github.com/langgenius/dify/blob/7753ba2d375beb0a5449042fbcbf5ade9a7cffb8/web/i18n/i18next-config.ts#L35

The rest part looks good to me, actually it's a pretty good one. Would you mind adding the missing ro-RO i18n files. Because it will failed to start.

Reply:
Sure, give me some time. :)

Reply:
I've added some translations. I will add some more in another PR to fast-forward this PR.